docs/docs/get-started/get-started.md

```diff
@@ -0,0 +1,82 @@
+---
+id: get-started
+title: Kurtosis Documentation
+slug: '/'
+sidebar_position: 1
+hide_table_of_contents: true
+---
+
+[Kurtosis](https://github.com/kurtosis-tech/kurtosis) is a tool for packaging and launching environments of containerized services where you want them, the way you want them, with one liners.
```

Comment:
I wonder if we can put your video in here

Reply:
100% doing this in next iteration